swagger.yaml

```diff
@@ -23,11 +23,69 @@ paths:
               schema:
                 type: string
 
+  /v1/users:
+    get:
+      operationId: listUsers
+      summary: Get list of Severa users.
+      description: Get list of Severa users.
+      tags:
+        - Users
+      parameters:
+        - name: isActive
+          in: query
+          description: Filter by users active state
+          schema:
+            type: boolean
+      responses:
+        "200":
+          description: Get list of Severa users
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/User"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  /v1/users/{uuid}:
+    get:
+      operationId: findUser
+      summary: Find user by keycloak id
+      description: Finds user by keycloak id in keycloak.
```

Comment:
Parameter named uuid with type UUID is a bit ambigious. Typically REST-endpoints are named /{version}/{resource in plural}/{identifier of resource}, with identifier being the Users id in this case. Or explicitly `{resource in singular}Id`, I'd call this `/v1/users/{userId}`.

Reply:
Resolved